### DISCUSSION TITLE
HikVision DS-KH6320

### DISCUSSION THREAD

**Discussion by TokarevSergey** [petrleocompel/hikaxpro_hacs#64] [Q&A]
Tell. Thanks to the project https://github.com/pergolafabio/Hikvision-Addons/tree/main/doorbell_beta I can change the scene on my device with the command:

![image](https://github.com/petrleocompel/hikaxpro_hacs/assets/98225056/be7d97c8-d707-4ff7-a999-9087e816f354)

Could you help and tell me which command can be used to disarm and arm the device. Thank you.

**Comment by petrleocompel** [comment]
1. KH6320 is not from AX family of products.

2. you are not reffering to our project at all you reffer to some other hikvision projects, so do not create issues. This is really an discussion question.

3. You can take a look in ISAPI Documentation - request should be 
![image](https://github.com/petrleocompel/hikaxpro_hacs/assets/9423543/811d435a-93f3-4fee-b73f-2f096ace2340)
`{}` is replaced by ID of Zone for  activation.
:warning: You need to have authorized session! :warning: 
Optional if you want to use code for activation (but it works without payload)
```json
{"Operate": {"moduleOperateCode": "1234"}}
```

or example for **AX Hybrid** "arm command"
![image](https://github.com/petrleocompel/hikaxpro_hacs/assets/9423543/a3450617-4579-4dd1-a03b-2fb653c7a81d)
